### ISSUE TITLE
Sample diversity

### ISSUE THREAD

**Issue by cbovar** [lucidrains/stylegan2-pytorch#45]
Hello!
Thank you for your work.

I have trained the model on FFHQ 1024x1024 dataset (parameters: `--batch-size 2 --gradient-accumulate-every 7 --network-capacity 16 --image-size 256`)

The output looks really good. However the same faces with little variation tend to appear several times (mode collapse)? It rarely generates Asian or Black faces for example.

Is there a way to increase this diversity ? Or is it just because of the dataset ? I was surprised to see a blue haired woman generated on[ thispersondoesnotexist.com](https://thispersondoesnotexist.com/) website, which would be very rare in most datasets.

![159-ema](https://user-images.githubusercontent.com/5098385/79305683-6d739e80-7f26-11ea-9b46-e09289fd11f1.jpg)

**Comment by lucidrains** [comment]
@cbovar thank you! that's the million dollar question! (how to prevent mode collapse, without hurting sample quality) i think the most accepted solution is PACGan https://arxiv.org/abs/1712.04086 , and some papers have combined it with StyleGAN, but with noticeable decrease in quality. perhaps I'll add it to the todos as something to experiment with in the future

**Comment by cbovar** [comment]
Alright :) I will close this issue since it's not related to this code.
